core/services/vrf/v2/listener_v2_log_listener.go

```diff
@@ -129,8 +129,14 @@ func (lsn *listenerV2) initializeLastProcessedBlock(ctx context.Context) (lastPr
 	}()
 
 	numBlocksToReplay := numReplayBlocks(lsn.job.VRFSpec.RequestTimeout, lsn.chain.ID())
-	ll.Debugw("running replay on log poller")
-	err = lp.Replay(ctx, mathutil.Max(latestBlock.FinalizedBlockNumber-numBlocksToReplay, 1))
+	replayStartBlock := mathutil.Max(latestBlock.FinalizedBlockNumber-numBlocksToReplay, 1)
+	ll.Debugw("running replay on log poller",
+		"numBlocksToReplay", numBlocksToReplay,
+		"replayStartBlock", replayStartBlock,
+		"requestTimeout", lsn.job.VRFSpec.RequestTimeout,
+		"finalizedLatestBlock", latestBlock.FinalizedBlockNumber,
```

Comment:
Would this already be added by line 123?

Reply:
right. let me remove the overlapping ones.